hivemind/server/__init__.py

```diff
@@ -113,8 +117,36 @@ def create(listen_on='0.0.0.0:*', num_experts=None, expert_uids=None, expert_cls
         # initialize experts
         if expert_uids is None:
             num_experts = num_experts if num_experts is not None else 1
```

Comment:
If it's None, better if we fail with an error here

Reply:
agreed, eta 1h